src/templates/slideshow/slideshow.scss

```diff
@@ -0,0 +1,46 @@
+.slideshow {
```

Comment:
can this be prefixed template, ie. .template-slideshow
clearly seperate the css from other css